### DISCUSSION TITLE
Is there a way to zoom out in the Song Canvas?

### DISCUSSION THREAD

**Discussion by Djard** [hydrogen-music/hydrogen#1672] [Hydrogen on Windows]
In my project, I have about 20 patterns and would like to view them all at once.  I am able to adjust the resolution and view the entire length of the project but not the height of the grid. In Full Screen, only one additional pattern track is visible. I dragged the lower border of the song canvas as far down as permitted. Is there a way to zoom out further, so that more pattern lines can be viewed in the song canvas?

**Comment by theGreatWhiteShark** [comment]
No, in its current implementation only the width of the grid can be altered.

**Comment by Djard** [comment]
Thank you.

**Comment by lacostenycoder** [comment]
@Djard in the meantime you [might use this ](https://unix.stackexchange.com/a/192896/152563)as a workaround if you're using Linux with Gnome
